src/create_methods.rb

```diff
@@ -1,73 +1,62 @@
-require_relative './person'
-require_relative './book'
-require_relative './rental'
-require_relative './student'
-require_relative './teacher'
-require_relative './classroom'
-
-class App
-  def list_all_books(db_books)
-    puts 'List of all books:'
-    db_books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
-  end
-
-  def create_book(db_books)
-    print 'Please enter the title of the book:'
-    title = gets.chomp
-    print 'Please enter the author of the book:'
-    author = gets.chomp
-    db_books.push(Book.new(title, author))
-    print 'Book created successfully'
-  end
-
-  def list_all_people(db_people)
-    puts 'List of all people:'
-    db_people.each { |person| puts "Name: #{person.name}, ID: #{person.id}, Age: #{person.age}" }
-  end
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
 
+class CreatorMethods
   def create_person(db_people)
-    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]'
-    option = gets.chomp
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp
+
     print 'Age: '
     age = gets.chomp
+
     print 'Name: '
     name = gets.chomp
 
-    case option
+    case person_type
     when '1'
       create_student(db_people, name, age)
     when '2'
       create_teacher(db_people, name, age)
     else
-      puts 'That is not a valid input'
+      puts 'Invalid option'
     end
   end
 
   def create_student(db_people, name, age)
-    puts 'Has parent permission? [Y/N]'
-    parent_permission = gets.chomp.downcase
-    student = ''
-    case parent_permission
-    when 'y'
-      student = Student.new(age, name, parent_permission: true)
-    when 'n'
-      student = Student.new(age, name, parent_permission: false)
-    else
-      puts 'That is not a valid input'
-      return
-    end
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp
+    parent_permission = parent_permission.downcase
+    parent_permission = parent_permission == 'y'
+
+    student = Student.new(age, name, parent_permission: parent_permission)
     db_people.push(student)
     puts 'Student created successfully'
   end
 
   def create_teacher(db_people, name, age)
     print 'Specialization: '
     specialization = gets.chomp
+
     teacher = Teacher.new(specialization, age, name)
     db_people.push(teacher)
     puts 'Teacher created successfully'
   end
 
+  def create_book(db_books)
+    print 'Title: '
+    title = gets.chomp
+
+    print 'Author: '
+    author = gets.chomp
+
+    book = Book.new(title, author)
+    db_books.push(book)
+    puts 'Book created successfully'
+  end
+
   def create_rental(db_data) # rubocop:todo Metrics/MethodLength
```

Comment:
- [ ] Fix :wrench:  Linter errors 


- [ ] Line 73, 81, 84. Please do not request inputs from the user in this method. This should have been done somewhere else(in a class or method) whose main purpose is for that. You can equally create different files with classes/methods to carry out that specific functionality.